.gitignore

```diff
@@ -1,6 +1,4 @@
-# VS Code
-.vscode
-
-# Custom
-bin/
-vendor/
+# Custom
+bin/
+vendor/
+nv2
```

Comment:
Why ignore `nv2`? This `nv2` binary is now in the `bin` folder.

Reply:
if someone still decides to run `go build` it will still end up as nv2 in the root.

Reply:
Then how about other binaries like `docker-nv2` and `docker-generate`? What if we have more later?

Reply:
Removed it.